pyseir/cli.py

```diff
@@ -50,42 +49,16 @@ def _generate_whitelist():
     gen.generate_whitelist()
 
 
-def _map_outputs(
-    state_regions: List[pipeline.Region],
-    output_interval_days=1,
-    states_only=False,
-    output_dir=None,
-    run_mode="default",
-):
-    assert all([r.is_state() for r in state_regions])
-    web_ui_mapper = WebUIDataAdaptorV1(
-        output_interval_days=output_interval_days, run_mode=run_mode, output_dir=output_dir,
-    )
-    for state in state_regions:
-        state_input = webui_data_adaptor_v1.RegionalInput.from_region(state)
-        web_ui_mapper.generate_state(
-            state_input, whitelisted_county_fips=[], states_only=states_only
-        )
-
-
 def _state_only_pipeline(
     region: pipeline.Region, run_mode=DEFAULT_RUN_MODE, output_interval_days=1, output_dir=None,
-):
+) -> model_fitter.ModelFitter:
```

Comment:
Can this return the infer rt, and ensembles results as well?

Reply:
It seems like those outputs could easily be passed into the WebUIInputRegion as well. but fine tackling that later

Reply:
Yes, my thought was get this merged with one output, then make another pass for the others.